test/models/pending_loan_test.rb

```diff
@@ -0,0 +1,3 @@
+require "test_helper"
+class PendingLoanTest < ActiveSupport::TestCase
+end
```

Comment:
I think there's one simple method in this model that should be tested.  Since it was already in the code before, I didn't add a test.